.github/actions/emit-metrics/action.yml

```diff
@@ -0,0 +1,74 @@
+name: Emit Metrics
+description: Emit metrics from another workflow/action using the kat tool
+inputs:
+  namespace:
+    description: The CloudWatch namespace in which to emit metrics
+    required: true
+  dimensions:
+    description: |-
+      The dimensions to include with emitted metrics, as a collection of `name=value` pairs. Multiple dimensions are
+      delimited by newlines (`\n`) and whitespace is trimmed from before/after each `name=value` pair. When passing
```

Comment:
Can this script also support space-delimited dimensions? I don't see a reason to _require_ delimiting with newlines

Reply:
I think the challenge in using spaces for delimiting is that dimension names/values can include spaces too. Parsing could then become ambiguous in situations like this:

```
Foo=Bar Operating System=Microsoft Windows Baz=Qux
```

It's not possible to determine whether `Foo=Bar` or `Foo=Bar Operating`, etc.

Do you foresee problems with newline delimited dimensions/metrics?

Reply:
No I think this should be fine, it just stood out to me on the initial review